.buildkite/pipeline.yml

```
@@ -18,5 +17,7 @@ steps:
     timeout: 100
 
   - label: "Run tests (bindists)"
-    command: ".buildkite/bindists-pipeline"
+    command: |
+      mkdir -p "$HOME/repo_cache"
+      docker run -it --network host -v $(pwd):/home/buildkite-agent/rules_haskell:rw -v $HOME/repo_cache:/home/buildkite-agent/repo_cache:rw -v $HOME/.cache/bazel:/home/buildkite-agent/.cache/bazel:rw -v $HOME/.stack:/home/buildkite-agent/.stack:rw --workdir /home/buildkite-agent/rules_haskell --user 1000 mrkkrp/rules_haskell:0.1.0 .buildkite/bindists-pipeline
```

```suggestion
      BINDIST_HOME=$HOME/bindist
      docker run -it --network host -v $(pwd):/home/buildkite-agent/rules_haskell:rw -v $BINDIST_HOME/repo_cache:/home/buildkite-agent/repo_cache:rw -v $BINDIST_HOME/.cache/bazel:/home/buildkite-agent/.cache/bazel:rw -v $BINDIST_HOME/.stack:/home/buildkite-agent/.stack:rw --workdir /home/buildkite-agent/rules_haskell --user 1000 mrkkrp/rules_haskell:0.1.0 .buildkite/bindists-pipeline
```
To avoid collisions between the nixpkgs and bindists pipelines, we should separate their stack and bazel caches. Not to have nix-store paths leak into the bindist pipeline or vice-versa.